docker-compose.yml

```diff
@@ -11,6 +11,33 @@ services:
     volumes:
       - pgdata:/var/lib/postgresql/data
     restart: unless-stopped
+    networks:
+      - backend
```

Comment:
Het hele `networks` stuk mag er normaliter geloof ik uit. Standaard krijg je een default netwerk.
Aan de andere kant: laat nu maar staan, want in *sommige* cornercases heb ik het wel eens toe moeten voegen :-)